Packs/Campaign/Scripts/GetSendEmailInstances/GetSendEmailInstances.py

```diff
@@ -0,0 +1,95 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+NO_SENDMAIL_INSTANCES_MSG = 'There is no enabled instances to send-mail'
+
+
+def get_all_integrations_commands():
+    """ Send API request with demisto rest api, to get all integration instances configured in the demisto. """
+    integration_commands_res = demisto.executeCommand("demisto-api-get", {"uri": "/settings/integration-commands"})
```

Comment:
note we recently added the `demisto.internalHttpRequest()` (same efficiency, a bit more intuitive to use and read)